io_scene_niftools/modules/nif_export/animation/transform.py

```diff
@@ -169,7 +192,7 @@ def export_kf_root(self, b_armature=None):
             # if variable_2:
             # controlledblock.set_variable_2(variable_2)
         else:
-            raise io_scene_niftools.utils.logging.NifError(f"Keyframe export for '{bpy.context.scene.niftools_scene.game}' is not supported.\nOnly Morrowind, Oblivion, Fallout 3, Civilization IV,"
+            raise io_scene_niftools.utils.logging.NifError(f"Keyframe export for '{bpy.context.scene.niftools_scene.game}' is not supported.\nOnly Morrowind, Oblivion, Skyrim, Fallout 3, Civilization IV,"
```

Comment:
Should we pull out the names of "supported" games to a list or something

Reply:
Would make sense!

Reply:
But technically we don't need to inform the user, do we?

Reply:
We can handle it